packaging/conda/meta.yaml

```diff
@@ -52,18 +52,27 @@ outputs:
         - {{ mpi }}
     files:
       - bin/MODL*
+      - bin/_getprocnumber*
+      - bin/khiops_env # [linux or osx]
+      - bin/khiops # [linux or osx]
+      - bin/khiops_coclustering # [linux or osx]
+      - bin/khiops_env.cmd # [win]
+      - bin/khiops.cmd # [win]
+      - bin/khiops_coclustering.cmd # [win]
     test:
       commands:
-        - MODL -s
-        - MODL_Coclustering -s
+        - khiops_env --env 
+        - khiops -s
+        - khiops_coclustering -s
```

Comment:
Why not stick to the MODL* binaries, as before?

Reply:
By using khiops, we are using mpi as configured in khiops_env.

Reply:
I am not sure I understand your comment (see also my reply above).